deploy/crds/upgrade.managed.openshift.io_v1alpha1_upgradeconfig_cr.yaml

```
@@ -3,6 +3,7 @@ kind: UpgradeConfig
 metadata:
   name: example-upgrade-config
 spec:
+  type: "OSD"
   desired:
     channel: "fast-4.4"
     force: false
```

Sorry to be a pain, should have caught this in Haoran's PR. Can we add the upgradeAt and proceed fields here. (upgradeAt is required for it to actually load)
```suggestion
  type: "OSD"
  upgradeAt: "2020-01-01T00:00:00Z"
  proceed: true
  desired:
    channel: "fast-4.4"
    force: false
```